packages/app-content-pages/src/screens/About/About.js

```diff
@@ -0,0 +1,136 @@
+import { Box, Heading } from 'grommet'
+import { useTranslation } from 'next-i18next'
+import styled from 'styled-components'
+
+import PageLayout from '../../shared/components/PageLayout/layout.js'
+import DropdownNav from '../../shared/components/DropdownNav/DropdownNav.js'
+import Head from '../../shared/components/Head'
+import Sidebar from '../../shared/components/Sidebar/Sidebar.js'
+import MaxWidthContent from '../../shared/components/MaxWidthContent/MaxWidthContent.js'
+import {
+  HeadingForNav,
+  mobileBreakpoint,
+  MobileHeading,
+  StyledGrid,
+  StyledHeading
+} from '../../shared/components/SharedStyledComponents/SharedStyledComponents.js'
+import OurMission from './components/OurMission.js'
+
+const StyledSidebar = styled(Sidebar)`
+  @media (width <= ${mobileBreakpoint}) {
+    display: none;
+  }
+`
+
+const StyledDropdownNav = styled(DropdownNav)`
+  @media (width > ${mobileBreakpoint}) {
+    display: none;
+  }
+`
+
+function HeadingForAboutNav({ sectionName, slug }) {
+  return (
+    <HeadingForNav
+      id={slug}
+      color={{ light: 'brand', dark: 'white' }}
+      level={2}
+      size='1.5rem'
+      tabIndex={-1}
+      textAlign='center'
+      style={{ padding: '30px 0 15px 0' }}
```

Comment:
This is sticking out to me, not necessarily an issue, but between Grommet, lib-react-components, and styled-components adding inline styling I think is not ideal, but I recognize sometimes necessary. Could the Grommet component this is eventually passed down to support the typical Grommet `pad` prop (it definitely [supports `margin`](https://v2.grommet.io/heading#margin))?

I noted this in Category.js and Team.js as well.

Reply:
Agreed it's not an ideal long term solution. As I'm building the new designs I'll continue to refactor `HeadingForNav` style props because there's a lot of variety in its spacing among different pages and sections. Once all sections are built, I'll convert to styled-components or a Grommet prop.